### PR TITLE
Update statement about declaring exceptions

### PR DESCRIPTION
**Pull request by melnikovi**
## Purpose of this pull request

Update statement about declaring exceptions.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5102) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5102) before we can accept your contribution.<br/>**4** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: keharper<br/>:white_check_mark: jeff-matthews<br/>:white_check_mark: sdzhepa<br/>:white_check_mark: dobooth<br/>:x: Lori Krell<hr/>**Lori Krell** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/coding-standards/technical-guidelines.md [hunk 1]
```diff
@@ -480,6 +480,8 @@ You need to read configuration from different sources (like database or filesyst
 
 5.18. `LocalizedException`s SHOULD be thrown in the presentation layer only.
 
+5.19. Each module or component MUST declare it's own exceptions. Exceptions declared in other components SHOULD NOT be thrown.
```
Comment (buskamuza) [inline guides/v2.2/coding-standards/technical-guidelines.md]: `it's` -> `its`

Comment (melnikovi) [reply guides/v2.2/coding-standards/technical-guidelines.md]: Resolved.

**Pull request merged**